### PR TITLE
pushed deets to the BE and FE readmes, and made the root readme pretty

### PR DESCRIPTION
**Pull request by Peedee2002**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,92 +1,38 @@
+<img src="https://img.shields.io/badge/-CSESoc-blue" />
+
 # Circles
+> see the [confluence workspace and links](https://compclub.atlassian.net/wiki/spaces/C/pages/1664385928/About+Circles) for further development documentation and a linktree to our jira boards, and more!
```
Comment (Joelrwest) [inline README.md]: A lot of these will be super nit picky grammar things, probs a capital here on 'see'

Comment (Joelrwest) [reply README.md]: Maybe 'for further development documentation, a linktree to our jira boards and more!' instead

File: README.md [hunk 2]
```diff
@@ -1,92 +1,38 @@
...
+## about Circles
+Tired of using a poorly laid out spreadsheet to cobble together a plan to follow for the next 3-8 years of your life?
+have no fear! Circles is here!
```
Comment (Joelrwest) [inline README.md]: 'Have no fear, Circles is here!'

File: README.md [hunk 3]
```diff
@@ -1,92 +1,38 @@
+<img src="https://img.shields.io/badge/-CSESoc-blue" />
+
 # Circles
+> see the [confluence workspace and links](https://compclub.atlassian.net/wiki/spaces/C/pages/1664385928/About+Circles) for further development documentation and a linktree to our jira boards, and more!
+## about Circles
```
Comment (Joelrwest) [inline README.md]: Capital

File: README.md [hunk 4]
```diff
@@ -1,92 +1,38 @@
...
+Tired of using a poorly laid out spreadsheet to cobble together a plan to follow for the next 3-8 years of your life?
+have no fear! Circles is here!
+
+Circles is a UNSW degree planner, which you can use to explore, check, and play around with your course planning.
+
+### features
```
Comment (Joelrwest) [inline README.md]: Capital

File: README.md [hunk 5]
```diff
@@ -1,92 +1,38 @@
+<img src="https://img.shields.io/badge/-CSESoc-blue" />
+
 # Circles
+> see the [confluence workspace and links](https://compclub.atlassian.net/wiki/spaces/C/pages/1664385928/About+Circles) for further development documentation and a linktree to our jira boards, and more!
+## about Circles
+Tired of using a poorly laid out spreadsheet to cobble together a plan to follow for the next 3-8 years of your life?
+have no fear! Circles is here!
+
+Circles is a UNSW degree planner, which you can use to explore, check, and play around with your course planning.
```
Comment (Joelrwest) [inline README.md]: Not sure how to reword this, maybe something like 'Circles is a UNSW degree planner where you can explore, validate and tinker with your course structure.', not sure about the with your course structure bit...

File: README.md [hunk 6]
```diff
@@ -1,92 +1,38 @@
...
+
+### features
+<img src="https://raw.githubusercontent.com/csesoc/Circles/dev/frontend/src/images/infographic.png">
+NOTE: we currently only support the Computer Science (3778) degree - but we will add more!
```
Comment (Joelrwest) [inline README.md:11]: This section isn't formatting as expected
image

File: README.md [hunk 7]
```diff
@@ -1,92 +1,38 @@
...
+- adaptive course selector
```
Comment (Joelrwest) [inline README.md]: Capital and maybe a colon on the top level dot point? Not sure

File: README.md [hunk 8]
```diff
@@ -1,92 +1,38 @@
...
+    - allows you to explore your currently unlocked courses as you do more of them
```
Comment (Joelrwest) [inline README.md]: Capital on each of these lower level dot points. Would also reword as 'Allows you to explore currently unlocked courses' and 'Shows all courses relevant to your degree'.

File: README.md [hunk 9]
```diff
@@ -1,92 +1,38 @@
...
+    - shows you all the courses relevant to your degree
+    - automatically checks if you meet the requirement to take a course
+
+- term planner
```
Comment (Joelrwest) [inline README.md]: Same deal in this section. Would reword as 'An intuitive drag and drop interface', 'Validates your progression to make sure you've met the requirements for future courses'

File: README.md [hunk 10]
```diff
@@ -1,92 +1,38 @@
...
+    - an intuitive drag and drop interface for planning terms
+    - validates your courses to make sure your future courses are unlocked before you take them
+    - allows you to generate a savable and sharable image of your plan
+
+### future
```
Comment (Joelrwest) [inline README.md]: Capital and same formatting stuff as previous section. Also no comma after expanding

File: README.md [hunk 11]
```diff
@@ -1,92 +1,38 @@
...
+this project is rapidly expanding, and our roadmap includes features such as
+- progression checker (NEW PAGE!)
+    - allows you to check how far you are into your degree
+    - lets you understand which requirements you have met and which you still need to complete
```
Comment (Joelrwest) [inline README.md]: I feel as though these are kinda the same dot point

**Review by Joelrwest — APPROVED** [review]

**Review by joooooooooooooooooooooooooooooooooooosh — APPROVED** [review]
